apps/src/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results.jsx

```diff
@@ -102,6 +116,43 @@ const LocalSummerWorkshopSurveyResults = React.createClass({
     );
   },
 
+  renderRow(row, i) {
+    let scoreCells;
+    let thisWorkshopData = this.state.thisWorkshop[row['key']];
+
+    if (this.state.facilitatorBreakdown && typeof thisWorkshopData === 'object') {
+      scoreCells = this.state.facilitatorNames.map((facilitator, i) => {
+        return (
+          <td key={i}>
+            {this.renderScore(row, thisWorkshopData[facilitator])}
+          </td>
+        );
+      });
```

Comment:
I'm okay making the key facilitator_name. If a workshop had two facilitators with identical names...well...other things would be broken now